### PR TITLE
Rename @adeira/eslint-config

### PR DESCRIPTION
**Pull request by mrtnzlml**

### CODE REVIEW

File: src/docs/blog/2019-06-21-relaxing-react-jsx-no-bind-eslint-rule.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ authorURL: https://github.com/mrtnzlml
 title: Relaxing react/jsx-no-bind eslint rule
 ---
 
-Recently, I significantly relaxed Eslint rule [`react/jsx-no-bind`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md) (in [`@kiwicom/eslint-config`](https://www.npmjs.com/package/@kiwicom/eslint-config) version 4.10.0). This article tries to explain what was this rule for and why I decided to change it.
+Recently, I significantly relaxed Eslint rule [`react/jsx-no-bind`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md) (in [`@adeira/eslint-config`](https://www.npmjs.com/package/@kiwicom/eslint-config) version 4.10.0). This article tries to explain what was this rule for and why I decided to change it.
```
Comment (tbergq) [inline src/docs/blog/2019-06-21-relaxing-react-jsx-no-bind-eslint-rule.md]: `https://www.npmjs.com/package/@kiwicom/eslint-config` -> `https://www.npmjs.com/package/@adeira/eslint-config`?

File: src/docs/docs/guides/eslint-config-migration.md [hunk 2]
```diff
@@ -82,6 +82,6 @@ The whole team should be committed to fixing the errors, don't make it to be a o
 
 # Think, have questions
 
-It's hard to have a motivation for this if you don't understand the purpose. If it's unclear for you why some rule was added into `@kiwicom/eslint-config` don't be afraid to ask on Slack in `#monorepo` channel.
+It's hard to have a motivation for this if you don't understand the purpose. If it's unclear for you why some rule was added into `@adeira/eslint-config` don't be afraid to ask on Slack in `#monorepo` channel.
```
Comment (tbergq) [inline src/docs/docs/guides/eslint-config-migration.md]: To ask in slack might not be valid anymore, create a github issue insteand maybe?

File: src/docs/blog/2019-06-21-relaxing-react-jsx-no-bind-eslint-rule.md (hunk 1 above)
Comment (mrtnzlml) [reply src/docs/blog/2019-06-21-relaxing-react-jsx-no-bind-eslint-rule.md]: Please rebase - this file doesn't exist anymore if I remember correctly.

File: src/docs/docs/guides/eslint-config-migration.md (hunk 2 above)
Comment (mrtnzlml) [reply src/docs/docs/guides/eslint-config-migration.md]: Docs need to be updated a lot - you can skip it.

File: src/docs/blog/2019-06-21-relaxing-react-jsx-no-bind-eslint-rule.md (hunk 1 above)
Comment (mrtnzlml) [reply src/docs/blog/2019-06-21-relaxing-react-jsx-no-bind-eslint-rule.md]: Haha, I asked you to rebase my PR. 🙈 I must've been tired... 😬 

On it.

Comment (tbergq) [reply src/docs/blog/2019-06-21-relaxing-react-jsx-no-bind-eslint-rule.md]: > Haha, I asked you to rebase my PR,

That is what I figured as well 😆

**Pull request merged**